src/modules/file-transfer-service/FileTransferService.ts

```diff
@@ -85,16 +75,24 @@ export class FileTransferService {
             throw new Error('File ID is missing');
         }
 
-        const config: AxiosRequestConfig = {
-            headers: this.prepareHeaders()
-        };
-
         return axios
-            .delete(`${this.url}/${fileId}`, config)
+            .delete(`${this.url}/${fileId}`, this.getAxiosConfig())
             .then(() => { return; })
             .catch(this.handleResponseError('deletion', fileId));
     }
 
+    private getAxiosConfig(headers: any = {}): AxiosRequestConfig {
+
+        return {
+            headers: {
+                ...this.prepareHeaders(),
+                ...headers
+            },
+            proxy: false
```

Comment:
Isn't it possible to drive it via proxy variables? Isn't it possible to use NO_PROXY variable for it?

Reply:
In this occasion, what's the advantage in doing this?

Reply:
Proxy is one of the things should be configurable not arbitrary set in software as settings differ from env to env.

Reply:
What do you suggest then? `constructor(private readonly url: string, private readonly key: string, useProxy: boolean)` something like this?

Reply:
Also responding to the first comment, I've researched on how it could be done and yes we could drive it using the NO_PROXY variable. By default it uses http_proxy

Reply:
Perfect, in that case no code change is needed, am I right? Usually good HTTP libraries support proxy env out of the box, I hope it is the same case here.

Reply:
I'm going to test this now, but hopefully no code changes just some documentation around that to add